spec/requests/custom_reactions_controller_spec.rb

```diff
@@ -121,21 +152,21 @@
     SiteSetting.post_undo_action_window_mins = 10
     sign_in(user_1)
     put "/discourse-reactions/posts/#{post_1.id}/custom-reactions/thumbsup/toggle.json"
-    expect(DiscourseReactions::Reaction.count).to eq(1)
-    expect(DiscourseReactions::ReactionUser.count).to eq(1)
-
-    put "/discourse-reactions/posts/#{post_1.id}/custom-reactions/thumbsup/toggle.json"
-    expect(DiscourseReactions::Reaction.count).to eq(0)
-    expect(DiscourseReactions::ReactionUser.count).to eq(0)
+    expect(DiscourseReactions::Reaction.count).to eq(2)
```

Comment:
for all this count check you should use the expect to change by xxx form, look at what we do in core for this